app/src/modules/content/components/version-menu.vue

```diff
@@ -137,7 +137,7 @@ function useRenameDialog() {
 		try {
 			const updates = {
 				key: newVersionKey.value,
-				...(newVersionName.value ? { name: newVersionName.value } : {}),
+				...(newVersionName.value !== currentVersion.value?.name ? { name: newVersionName.value } : {}),
```

Comment:
I know that has already been there, but could be
```suggestion
				...(newVersionName.value !== currentVersion.value?.name && { name: newVersionName.value }),
```
right?

Reply:
Yes! Nice optimisation! Thanks~